ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/util/SlotAndBlockRootAndBlobIndex.java

```diff
@@ -45,6 +45,10 @@ public UInt64 getBlobIndex() {
     return blobIndex;
   }
 
+  public boolean hasNoBlobs() {
```

Comment:
I'd put a `static Uint64 NO_BLOBS_INDEX = UInt64.MAX_VALUE;`
and call this method `isNoBlobsKey()`